src/github/github.ts

```diff
@@ -136,6 +137,21 @@ export class Github {
   }
 
   /**
+   * Given an extracted or cloned github directory, unlink files that are not
+   * intended to exist in the template and serve other purposes in the gith
+   * repository.
+   */
+  removeUnwantedFiles(outDir: string) {
+    [
```

Comment:
can you move this list of files into a constant above? It will be easier to re-configure if we want to add/remove files in the future.